package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "node-sass",
   "version": "3.3.2",
-  "libsass": "3.2.5",
+  "libsass": "3.3.0",
```

Comment:
I want to merge this now :)

https://github.com/sass/libsass/releases/tag/3.3.0-beta2